[2014-06-25 09:07:07] log.INFO: Accessing postUpload @ UploadController [] []
[2014-06-25 09:07:07] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function getClientOriginalName() on a non-object' in /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php:26 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-06-25 09:08:12] log.INFO: Accessing postUpload @ UploadController [] []
[2014-06-25 09:53:01] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' with message 'Controller method not found.' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php:290 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(302): Illuminate\Routing\Controllers\Controller->missingMethod(Array) #1 [internal function]: Illuminate\Routing\Controllers\Controller->__call('uploadExce', Array) #2 [internal function]: UploadController->uploadExce() #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('uploadExce', Array) #5 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'uploadExce', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #13 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #14 {main} [] []
[2014-06-25 09:57:39] log.ERROR: exception 'Illuminate\Database\Eloquent\MassAssignmentException' with message 'client_number' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:310 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(237): Illuminate\Database\Eloquent\Model->fill(Array) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(375): Illuminate\Database\Eloquent\Model->__construct(Array) #2 /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php(80): Illuminate\Database\Eloquent\Model::create(Array) #3 [internal function]: UploadController->uploadExcel() #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #5 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('uploadExcel', Array) #6 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'uploadExcel', Array) #7 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #8 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #9 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #10 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #11 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #12 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #13 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #14 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #15 {main} [] []
[2014-06-25 09:58:12] log.ERROR: exception 'Illuminate\Database\Eloquent\MassAssignmentException' with message 'client_number' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:310 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(237): Illuminate\Database\Eloquent\Model->fill(Array) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(375): Illuminate\Database\Eloquent\Model->__construct(Array) #2 /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php(80): Illuminate\Database\Eloquent\Model::create(Array) #3 [internal function]: UploadController->uploadExcel() #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #5 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('uploadExcel', Array) #6 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'uploadExcel', Array) #7 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #8 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #9 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #10 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #11 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #12 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #13 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #14 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #15 {main} [] []
[2014-06-25 10:02:22] log.ERROR: exception 'ErrorException' with message 'Undefined variable: rows' in /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php:84 Stack trace: #0 /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php(84): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/opt/lampp/htdo...', 84, Array) #1 [internal function]: UploadController->uploadExcel() #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('uploadExcel', Array) #4 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'uploadExcel', Array) #5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #6 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #7 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #8 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #9 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #10 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #11 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #12 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #13 {main} [] []
[2014-06-25 11:40:23] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '$xls_fields' (T_VARIABLE)' in /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php:41 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-06-25 11:41:27] log.INFO: Uploading excel file -> test.xlsx [] []
[2014-06-25 11:41:27] log.INFO: File uploaded -> test.xlsx :: Processing started... [] []
[2014-06-25 11:41:27] log.INFO: File is readable -> test.xlsx [] []
[2014-06-25 11:41:27] log.INFO: File has 10 records [] []
[2014-06-25 11:41:28] log.INFO: Processing of file completed [] []
[2014-06-25 11:42:30] log.INFO: Uploading excel file -> About Me.docx [] []
[2014-06-25 11:42:30] log.INFO: File uploaded -> About Me.docx :: Processing started... [] []
[2014-06-25 11:42:30] log.INFO: File is readable -> About Me.docx [] []
[2014-06-25 11:42:31] log.ERROR: exception 'ErrorException' with message 'DOMDocument::loadHTMLFile(): Invalid char in CDATA 0x3 in uploads/About Me.docx, line: 1' in /opt/lampp/htdocs/noyawa/vendor/phpoffice/phpexcel/Classes/PHPExcel/Reader/HTML.php:427 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleError(2, 'DOMDocument::lo...', '/opt/lampp/htdo...', 427, Array) #1 /opt/lampp/htdocs/noyawa/vendor/phpoffice/phpexcel/Classes/PHPExcel/Reader/HTML.php(427): DOMDocument->loadHTMLFile('uploads/About M...', 12) #2 /opt/lampp/htdocs/noyawa/vendor/phpoffice/phpexcel/Classes/PHPExcel/Reader/HTML.php(141): PHPExcel_Reader_HTML->loadIntoExisting('uploads/About M...', Object(PHPExcel)) #3 /opt/lampp/htdocs/noyawa/vendor/phpoffice/phpexcel/Classes/PHPExcel/IOFactory.php(192): PHPExcel_Reader_HTML->load('uploads/About M...') #4 /opt/lampp/htdocs/noyawa/app/controllers/UploadController.php(33): PHPExcel_IOFactory::load('uploads/About M...') #5 [internal function]: UploadController->uploadExcel() #6 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #7 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('uploadExcel', Array) #8 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'uploadExcel', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #16 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #17 {main} [] []
[2014-06-25 12:46:58] log.INFO: Accessing postClient @ ClientController [] []
[2014-06-25 12:49:36] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /opt/lampp/htdocs/noyawa/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /opt/lampp/htdocs/noyawa/public/index.php(49): Illuminate\Foundation\Application->run() #5 /opt/lampp/htdocs/noyawa/server.php(19): require_once('/opt/lampp/htdo...') #6 {main} [] []
